apps/compound/src/config.ts

```diff
@@ -21,17 +20,16 @@ export type TokenItem = {
 
 export const rpc_token = process.env.REACT_APP_RPC_TOKEN || '';
 
-export const getTokenList = (network: Networks): Array<TokenItem> => {
-  const lowercaseNetwork = network.toLowerCase();
+export const getTokenList = (chainId: number): Array<TokenItem> => {
 
-  if (lowercaseNetwork !== 'rinkeby' && lowercaseNetwork !== 'mainnet') {
-    throw Error(`Not supported Network ${network}`);
+  if (chainId !== 4 && chainId !== 1) {
```

Comment:
Let's not use 4, 1 etc as literal numbers. They should come from an enum. Like CHAINS.RINKEBY.